packages/db/src/system.ts

```diff
@@ -47,10 +47,12 @@ export type ConnectOptions = Meta.ConnectOptions<Collections>;
 
 const system = Meta.forDefinitions(definitions);
 
-export const connect: (options: TruffleConfig | ConnectOptions) => Db =
-  system.connect;
+export const connect: (
+  options: TruffleConfig | ConnectOptions | undefined
```

Comment:
More normally, @truffle/db code does this sort of thing this way:

```suggestion
  options?: TruffleConfig | ConnectOptions
```

Reply:
Ah yes, this is what I wanted.